src/qibolab/platforms/multiqubit.py

```diff
@@ -45,7 +53,7 @@ def reload_settings(self):
         super().reload_settings()
         self.characterization = self.settings["characterization"]
         self.qubit_channel_map = self.settings["qubit_channel_map"]
-        self.nshots = self.settings["settings"]["nshots"]
+        self.nshots = self.settings["settings"]["hardware_avg"]
```

Comment:
`hardware_avg` was agreed to be renamed to nshots so don't revert the changes

Reply:
Sorry, I did not pushed the latest changes where the hardware_avg was changed to nshots. Now it should be ok.